constitution.md

```diff
@@ -0,0 +1,232 @@
+
+Other responsibilities include: admitting (and if necessary expelling) projects,
+organizing community meetings/conferences, resolving disputes between projects,
+allocating funds, clarifying procedures for each membership type, and publicizing
+the Chisel community.
+
+Projects are admitted (or expelled) by a 2/3 vote of the TAC.
+A list of current projects is maintained in a document in this repository.
+
+The TAC is made up of members of the community as well as one member appointed by
+the CHIPS Alliance Board.
+The members are listed by name and Github username in a document within
+this repository. Membership change is reflected by pull requests
+against the membership list document. Members have the power to remove themselves.
+Membership additions (or expulsions) require 2/3 approval of the current members
+of the TAC.
+
+#### TAC Chair
+The chair of the TAC is selected from and by the TAC members by popular vote.
+The chair's responsibilities are to schedule and run meetings of the TAC, as well
+as count votes and enforce voting periods for any action that requires a vote.
+
+The chair will serve until they step down or are replaced by a new vote of the TAC.
+Chair elections generally occur during scheduled meetings, but in the event of an
+absent or otherwise misbehaving chair, such a vote can be initiated by any TAC member
+as a PR against the TAC membership document.
+
+### Administrator
+Consists of one lead and any number of additional administrators who direct a
+given project and report progress to the TAC
+They have the responsibility to represent the user community’s diverse set of
+interests.
+
+Administrators determine how underlying projects are managed, which may include
+enforcing collaboration policies, and determining how contributors join the project.
+They also resolve disagreements between Reviewers, Developers, Volunteers, and users.
+
+Administrators have all of the rights and responsibilities of Reviewers.
+
+### Reviewers
+Reviewers are selected per project repository, and their primary responsibility is to maintain the health of the code 
+repositories by enforcing a consistent coding style and good software engineering practices. Their second responsibility 
+is to encourage community growth by mentoring enthusiastic contributors.
+
+*Reviewers will be the members who most directly interact with the developer community, so it is of primary importance that 
+they encourage a welcoming atmosphere.*
+
+Reviewers are the only members who have write-privileges to their repository, so they must be active regularly such that 
+development will not stall on their reviews. The procedure for becoming a Reviewer is outlined in the following Developers 
+section.
+
+Reviewers strictly subsume Developers (e.g. they also have all the rights and responsibilities of Developers).
+
+### Developers
+Developers are the most common membership type, and is an entry point for enthusiastic community members. They have voting
```

Comment:
Has this ever happened / has this happened yet?

Reply:
No, primarily because we haven't had a constitution yet. Do you have an objection to this idea?